SSA/Projects/InstCombine/ForLean.lean

```diff
@@ -37,19 +37,19 @@ lemma two_pow_pred_mod_eq_two_pred (h : w > 0): 2 ^ (w - 1) % 2 ^ w = 2 ^ (w - 1
 
 namespace BitVec
 
-def ushr_xor_distrib (a b c : BitVec w) :
+lemma ushr_xor_distrib (a b c : BitVec w) :
     (a ^^^ b) >>> c = (a >>> c) ^^^ (b >>> c) := by
   simp only [HShiftRight.hShiftRight]
   ext
   simp
 
-def ushr_and_distrib (a b c : BitVec w) :
+lemma ushr_and_distrib (a b c : BitVec w) :
     (a &&& b) >>> c = (a >>> c) &&& (b >>> c) := by
   simp only [HShiftRight.hShiftRight]
   ext
   simp
 
-def ushr_or_distrib (a b c : BitVec w) :
+lemma ushr_or_distrib (a b c : BitVec w) :
     (a ||| b) >>> c = (a >>> c) ||| (b >>> c) := by
```

Comment:
These changes are unrelated and should not be included here.

Reply:
Yes, you are right.

Reply:
I want these changes to disappear. 😄

Reply:
Now these are back again. The last revert was not really needed, I think.

Reply:
Try to work with @AtticusKuhn to clean up this PR. You may need some help with checking how to make this clean and ready.